test/ziggurat/util/rabbitmq.clj

```diff
@@ -51,6 +51,12 @@
         queue-name (prefixed-channel-name topic-name channel-name queue-name)]
     (get-msg-from-rabbitmq queue-name topic-name)))
 
+(defn get-message-from-channel-retry-queue [topic channel sequence]
+  (let [{:keys [queue-name]} (:delay (rabbitmq-config))
+        delay-queue-name (producer/delay-queue-name (rutil/with-channel-name topic channel) queue-name)
```

Comment:
will the queue names not have `_3` or something attached to it? is it only the exchanges?

Reply:
each exchange will have different queue, exchange with `_3` will bind to queue with `_3`